security/auto-host-endpoints.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Protect Kubernetes nodes
+description: Protect Kubernetes nodes with host endpoints managed by {{site.prodname}}
+---
+
+### Big picture
+
+Secure Kubernetes nodes with host endpoints managed by {{site.prodname}}.
+
+### Value
+
+{{site.prodname}} can automatically create host endpoints for your Kubernetes nodes. The lifecycle of these host endpoints are managed by {{site.prodname}} in response to an evolving cluster to ensure policy protecting your nodes is always enforced.
+
+### Features
+
+This how-to guide uses the following Calico features:
+- **HostEndpoint**
+- **Profile**
+- **KubeControllersConfiguration**
+- **GlobalNetworkPolicy**
+
+### Concepts
+
+### Host endpoints
+
+Each host has one or more network interfaces that it uses to communicate externally. You can use {{site.prodname}} network policy to secure these interfaces (called host endpoints).
```

Comment:
You can represent these interfaces in Calico using host endpoints and then use network policy to secure them

Reply:
Thanks :) 7e6501ca8e4fcedf6cfb63a4d6f91098cbc564ff